### ISSUE TITLE
Heroku deployment is down!

### ISSUE THREAD

**Issue by davidbludlow** [saveourtool/diktat-demo#66]
As of 2022-12-6, when I go to https://ktlint-demo.herokuapp.com/ all I see is
![image](https://user-images.githubusercontent.com/5985136/205989894-117ae4cf-0b89-41e8-a3ba-5334339504e5.png)

**Comment by akuleshov7** [comment]
@davidbludlow hey, thank you for the report! Yes, Herokku is not free anymore :(
So we are moving this instance to https://saveourtool.com/#/demo/diktat

**Comment by paul-dingemans** [comment]
> @davidbludlow hey, thank you for the report! Yes, Herokku is not free anymore :( So we are moving this instance to https://saveourtool.com/#/demo/diktat
> 
> After the work will be finished, we will update ktlint documentation

That would be great. We indeed got a report about a dead link ;-)

**Comment by akuleshov7** [comment]
> That would be great. We indeed got a report about a dead link ;-)

Yeah, we are testing it right now - looks more stable, but we also need to make some extra page for ktlint;
BTW: if you have any suggestions of how we can improve demo - you are welcome - will try to enhance :)
https://saveourtool.com/#/demo/diktat

**Comment by davidbludlow** [comment]
The link on https://pinterest.github.io/ktlint/install/overview/ is still to the broken deployment

**Comment by paul-dingemans** [comment]
> The link on https://pinterest.github.io/ktlint/install/overview/ is still to the broken deployment

Updated via https://github.com/pinterest/ktlint/pull/1782
